js-api-doc/value/color.d.ts

```diff
@@ -40,26 +40,24 @@ export class SassColor extends Value {
 
   get alpha(): number;
 
-  changeRgb(options: {
+  change(options: {
```

Comment:
The idiomatic way to write this is also to use a union rather than overloads (btw, that's how you did it in embedded-host-node)